.github/workflows/deploy.yml

```diff
@@ -0,0 +1,34 @@
+name: Deploy
+
+# Run deploy only on published releases.
+on:
+  release:
+    types: [published]
+
+jobs:
+
+  deploy:
+    name: Deploy to WordPress.org
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: WordPress Plugin Deploy
+        id: deploy
+        uses: 10up/action-wordpress-plugin-deploy@stable
+        with:
+          generate-zip: true
+        env:
+          #SVN_USERNAME: ${{ secrets.SVN_USERNAME }}
+          #SVN_PASSWORD: ${{ secrets.SVN_PASSWORD }}
```

Comment:
For added security, I've commented out the secrets. Once the pull request is ready to be merged, I will remove the comments.